GlobalViolence/R/DataPrep_GBD.R

```diff
@@ -11,34 +11,34 @@ dir.create(here("GlobalViolence","Data","Grouped","GBD"), showWarnings = FALSE,
 GBD     <- readRDS(file.path(gbd.folder,"GBD.rds"))
 
 # remove Percent, not needed now.
-GBD <- GBD[metric_name != "Percent"]; gc()
```

Comment:
Hi!
I think you're working with a version of the GBD data that you already had?
In the DownloadData.R script there's a permalink which I've just verified still works. It leads to a page where everything is pre-selected, and you just need to click to download the csvs. Then you end up with a list of N csv files with urls that you can then enter into the script. This is lines 111-177. It might be possible to download names without the `_name` part to them, but then I'd need to manually re-write this whole section, so I'd rather not take the code changes that eliminate `_name`

Reply:
ok, great, sorry for that : \

Reply:
of course. cool that we have a permalink. I thought it expired and we had to download it each time anyway, so I didn´t even check it.